src/style/price.css

```diff
@@ -0,0 +1,22 @@
+.product__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
```

Comment:
I think you also need to use align-items: center
![image](https://user-images.githubusercontent.com/54936044/221815821-6faa9e84-0f63-4c2a-8b87-0d18e14e0d2a.png)